### PR TITLE
Random emergency shuttle time

### PR DESCRIPTION
**Pull request by metalgearsloth**
60 to 180 seconds. Rounds up to nearest 10.
All other FTL will go to the default of 30s.

Renamed shuttle.emergency_enabled to shuttle.emergency for consistency with cargo and arrivals.

Merge after https://github.com/space-wizards/space-station-14/pull/10043

:cl:
- tweak: Emergency shuttle FTL time is now randomised from 60 to 180 seconds.

### CODE REVIEW

File: Content.Server/Shuttles/Systems/ShuttleSystem.EmergencyConsole.cs [hunk 1]
```diff
@@ -89,15 +100,21 @@ private void SetAuthorizeTime(float obj)
...
+    private void SetMinTransitTime(float obj)
     {
-        TransitTime = obj;
+        MinimumTransitTime = obj;
+    }
+
+    private void SetMaxTransitTime(float obj)
+    {
+        MaximumTransitTime = obj;
     }
```
Comment (Zumorica) [inline Content.Server/Shuttles/Systems/ShuttleSystem.EmergencyConsole.cs:111]: This should probably handle max < min sanity here

**Comment by PJB3005** [comment]
Why? Being able to rely on the emergency shuttle taking a fixed time seems like a good game mechanic to me.

**Comment by metalgearsloth** [comment]
> Why? Being able to rely on the emergency shuttle taking a fixed time seems like a good game mechanic to me.

Beats me, I've been told it was too long at 3 minutes, then too short at 1 minute, then got told random.

**Comment by PJB3005** [comment]
That doesn't sound like a very good balancing strategy. What's the recall period? Being able to just abruptly end the game if no head of staff makes it to a comms console in *60 seconds* is absurd (and that's assuming the shuttle can always be recalled. If it's like SS13 where it can only be recalled halfway it's even worse).

**Comment by metalgearsloth** [comment]
> That doesn't sound like a very good balancing strategy. What's the recall period? Being able to just abruptly end the game if no head of staff makes it to a comms console in _60 seconds_ is absurd (and that's assuming the shuttle can always be recalled. If it's like SS13 where it can only be recalled halfway it's even worse).

This is the time it takes to get to centcomm after it's launched (the call time is minimum 5 minutes and the dock time is 3 minutes for reference which aren't affected).

**Comment by ike709** [comment]
Random is cringe

**Comment by mirrorcult** [comment]
not really sure i understand the motivation here still

**Comment by Emisse** [comment]
sad

**Comment by metalgearsloth** [comment]
emisse gon kil me

**Pull request merged**